### ISSUE TITLE
quick feature request: Address / location field in Group links

### ISSUE THREAD

**Issue by trip-nine** [DiscipleTools/disciple-tools-bulk-magic-link-sender#63]
Hi team!
quick feature request - can you please add the Address/location field in Group links?

This will allow us to plot where new groups are in the field from just a magic link! 

will be a HUGE value for field workers

<img width="385" alt="image" src="https://user-images.githubusercontent.com/37304634/194622824-ef1be133-0942-4cd8-854e-353e056a915d.png">

**Comment by cairocoder01** [comment]
@corsacca Yes, that field still needs some development in the components. I think Micah did some work on it, but I don't know how fully tested it is. It probably needs some backend integration as well since I'm not sure how the API works for those fields.

**Comment by corsacca** [comment]
@micahmills ?

**Comment by micahmills** [comment]
I haven't done much with it, but it is a top priority for the components.
